master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -172,7 +172,6 @@ private[celeborn] class Master(
   private var checkForApplicationTimeOutTask: ScheduledFuture[_] = _
   private var checkForUnavailableWorkerTimeOutTask: ScheduledFuture[_] = _
   private var checkForHDFSRemnantDirsTimeOutTask: ScheduledFuture[_] = _
-  private var checkForS3RemnantDirsTimeOutTask: ScheduledFuture[_] = _
```

Comment:
This is not needed anymore, as the task was combined for `hasHDFSStorage` and `hasS3Storage`